_dev/back/index.js

```diff
@@ -153,15 +154,22 @@ $(document).ready(() => {
           this.running = false;
           if (res.result) {
             this.cursor = 0;
+            totalCount = 0;
             $('#indexing-warning').hide();
             $(this).html(this.legend);
             $('#ajax-message-ok span').html(translations.price_indexation_finished);
             $('#ajax-message-ok').show();
             return;
           }
 
+          totalCount += parseInt(res.count, 10);
           this.cursor = parseInt(res.cursor, 10);
-          $(this).html(this.legend + translations.price_indexation_in_progress.replace('%s', res.count));
+          $(this).html(
+            this.legend + translations.price_indexation_in_progress.replace(
```

Comment:
Isn't eslint throwing an error about this chain?

Reply:
I was also surprised that eslint says "ok"